src/it/java/com/google/checkstyle/test/chapter5naming/rule528typevariablenames/MethodTypeParameterNameTest.java

```diff
@@ -38,7 +38,7 @@ protected String getPackageLocation() {
         return "com/google/checkstyle/test/chapter5naming/rule528typevariablenames";
     }
 
-    @BeforeAll
+    @BeforeEach
     public static void setConfigurationBuilder() throws CheckstyleException {
```

Comment:
BeforeEach cannot be static. I am not sure if pitest is actually running the method because of this but it is causing a CI failure.